### PR TITLE
16.0 mig contract mandate

### PR DESCRIPTION
**Pull request by aiendry-aktivsoftware**
Standard migration to version 16.0

Depends on  [16.0] [MIG] contract_payment_mode #947

### CODE REVIEW

**Comment by rousseldenis** [comment]
@aiendry-aktivsoftware FYI, you can embed depending PR's in order to easy review and tests running : https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29

File: contract_mandate/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Contract Mandate",
     "summary": "Mandate in contracts and their invoices",
-    "version": "14.0.1.0.0",
+    "version": "16.0.1.0.0",
     "author": "Odoo Community Association (OCA), " "Tecnativa",
```
Comment (FrancoMaxime) [inline contract_mandate/__manifest__.py:8]: Hi, can you fix the typo created by an old pre-commit ?

```suggestion
    "author": "Odoo Community Association (OCA), Tecnativa",
```

**Review by FrancoMaxime — APPROVED** [review]
LGTM - code review

1 minor comment

**Comment by florian-dacosta** [comment]
@aiendry-aktivsoftware @rousseldenis Possible to reopen this PR please ?

**Pull request closed without merging**